pages/dkp/kommander/2.2/release-notes/2.2.0/index.md

```diff
@@ -278,7 +278,7 @@ Upgrading catalog applications using Spark Operator can fail when running `dkp u
     metadata:
     name: spark-operator
     annotations:
-        "helm.sh/hook": pre-install
+        "helm.sh/hook": pre-install, pre-upgrade
```

Comment:
We actually don’t need the hook annotations here, they won’t apply. They are only used by helm

Reply:
you are right -- i tested that this hooks annotations only take effect upon install/upgrade, but they do need to exist to be counted as hook resource and will be counted upon future upgrades. 

I added to the ClusterRoleBinding to be consistent with the ClusterRole.